FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/DemoActivity.kt

```diff
@@ -18,7 +18,6 @@ abstract class DemoActivity : AppCompatActivity() {
     companion object {
         const val DEMO_ID = "demo_id"
     }
-
```

Comment:
nit : can we have a last pass too not include extra changes? like line removal

Reply:
Done. Fixed in next patch